libcxx/test/std/containers/sequences/vector.bool/flip.pass.cpp

```diff
@@ -0,0 +1,54 @@
+//===----------------------------------------------------------------------===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+// <vector>
+
+// void flip();
+
+#include <cassert>
+#include <memory>
+#include <vector>
+
+#include "min_allocator.h"
+#include "test_allocator.h"
+#include "test_macros.h"
+
+template <typename Allocator = std::allocator<bool> >
```

Comment:
```suggestion
template <typename Allocator>
```
I'd rather make this explicit. It's not like we gain much by defaulting this.

Reply:
Thank you for your detailed review and insightful comments! I really appreciate your feedback.